codecov_auth/commands/owner/interactors/tests/test_save_terms_agreement.py

```diff
@@ -28,45 +28,39 @@ def execute(
         )
 
     @freeze_time("2022-01-01T00:00:00")
-    def test_update_owner_profile_when_agreement_is_false(self):
+    def test_update_user_when_agreement_is_false(self):
         self.execute(current_owner=self.current_owner, input={"termsAgreement": False})
         before_refresh_business_email = self.current_owner.business_email
 
-        owner_profile: OwnerProfile = OwnerProfile.objects.filter(
-            owner=self.current_owner
-        ).first()
-        assert owner_profile.terms_agreement == False
-        assert owner_profile.terms_agreement_at == timezone.datetime(2022, 1, 1)
+        user: User = User.objects.filter(id=self.current_owner.user_id).first()
+        assert user.terms_agreement == False
+        assert user.terms_agreement_at == timezone.datetime(2022, 1, 1)
 
         self.current_owner.refresh_from_db()
         self.current_owner.business_email == before_refresh_business_email
 
     @freeze_time("2022-01-02T00:00:00")
-    def test_update_owner_profile_when_agreement_is_true(self):
+    def test_update_user_when_agreement_is_true(self):
         self.execute(current_owner=self.current_owner, input={"termsAgreement": True})
         before_refresh_business_email = self.current_owner.business_email
 
-        owner_profile: OwnerProfile = OwnerProfile.objects.filter(
-            owner=self.current_owner
-        ).first()
-        assert owner_profile.terms_agreement == True
-        assert owner_profile.terms_agreement_at == timezone.datetime(2022, 1, 2)
+        user: User = User.objects.filter(id=self.current_owner.user_id).first()
+        assert user.terms_agreement == True
+        assert user.terms_agreement_at == timezone.datetime(2022, 1, 2)
 
         self.current_owner.refresh_from_db()
         self.current_owner.business_email == before_refresh_business_email
 
     @freeze_time("2022-01-03T00:00:00")
-    def test_update_owner_and_profile_when_email_isnt_empty(self):
+    def test_update_owner_and_user_when_email_isnt_empty(self):
```

Comment:
nitpick: change `isnt` to `is_not`. I originally created the test with `isnt`, so not your doing, but I recently installed this extension, https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker, that highlights typos, so in the spirit of slowly moving our codebase to proper names

Reply:
Done. Cool tool, installed too.